docs/integrations.md

```diff
@@ -1,272 +1,241 @@
-# Integrations
+# Editor Integrations
+
+## Ruff Language Server (`ruff server`)
+
+The Ruff language server, also known as `ruff server`, is what powers the diagnostic and formatting capabilities of Ruff's VS Code extension and other editors. It is a single, common backend for editor integrations built directly into Ruff, and a direct replacement for `ruff-lsp`, our previous language server. You can read more about `ruff server` in the [`v0.4.5` blog post](https://astral.sh/blog/ruff-v0.4.5).
+
```

Comment:
I would remove the mention of `ruff-lsp` from here. I worry that it is more confusing than helpful for users that just want to set up their IDE. 

We could even argue that for most users, the entire *Ruff Language Server* section isn't relevant if, all they want to do, is set up their VS code.

Reply:
I think the added context about the language server would be helpful for users using editors besides VS Code. I'm open to removing the mention of `ruff-lsp` here, but I think we should still document it in its deprecated state.

Reply:
That makes sense. To me, the mentioning in

> Other editors may support the legacy `ruff-lsp` server; see the [`ruff-lsp` documentation](https://github.com/astral-sh/ruff-lsp#setup) for more.

seems sufficient.